netlink-packet-route/src/rtnl/link/nlas/mod.rs

```diff
@@ -409,7 +412,11 @@ impl<'a, T: AsRef<[u8]> + ?Sized> ParseableParametrized<NlaBuffer<&'a T>, u16> f
             IFLA_CARRIER_UP_COUNT => CarrierUpCount(payload.to_vec()),
             IFLA_CARRIER_DOWN_COUNT => CarrierDownCount(payload.to_vec()),
             IFLA_NEW_IFINDEX => NewIfIndex(payload.to_vec()),
-            IFLA_PROP_LIST => PropList(payload.to_vec()),
+            IFLA_PROP_LIST => {
+                let err = "invalid IFLA_PROP_LIST value";
+                let buf = NlaBuffer::new_checked(payload).context(err)?;
+                PropList(VecPropList::parse(&buf).context(err)?.0)
```

Comment:
Instead of using `VecPropList`, you may use 

```rust
for nla in NlasIterator::new(payload()) {
    let nla = &nla.context(error_msg)?;
    let parsed = PropList::parse(nla).context(error_msg)?;
    nlas.push(parsed);
}
PropList(nlas)
```

Reply:
I do the same as AfSpecInet and remove VecPropList